90.md

```diff
@@ -0,0 +1,75 @@
+NIP-90
+======
+
+Partially Signed Bitcoin Transactions
+--------------------------------------------
+`draft` `optional` `author:ronaldstoner` `author:Armin Sabouri` `author:satsie` 
+
+# Abstract
+
+This NIP defines an event type for partially signed bitcoin transactions (PSBT) that can be transferred via the Nostr relay network. `1174` is the event kind representing a Bitcoin transaction in standard PSBT format, regardless if the transaction is partially signed or fully finalized.
+
+A special event with kind `1174` which contains an encoded PSBT. PSBTs used under this spec are version 0 [BIP-0174](https://github.com/bitcoin/bips/blob/master/bip-0174.mediawiki).
+
+PSBTs are encoded as a base64 string, and are contained in the `content` field. These events use a specific kind identifier so that clients do not need to render PSBTs the same as they do notes, invoices, and other event kind identifiers. If clients wish to display PSBTs they can choose to do so in whatever format they wish (string, QR Code, view render).
+
+The network magic is contained in the `tags` field. This is to identify the network that the transaction is for. The [common ones](https://en.bitcoin.it/wiki/Protocol_documentation#Message_structure) are:
+```
+- mainnet: f9beb4d9
+- testnet: 0b110907
+- signet: 0a03cf40
+- regtest: fabfb5da
+``` 
+
+PSBTs transmitted via the Nostr network provide a decentralized communication layer, encryption, and digital signature benefits with no username and password required for participants to coordinate and sign transactions using separate sovereign bitcoin private keys.
+
+# Protocol Flow
+The following are two protocol flow examples using the 1174 event kind. Other flows that could be used encompass various interactive/multi-party solutions such as coin joins, swaps or Schnorr bases multisig solutions (FROST/Musig2).
+
+## Multisig P2WSH
+2-of-2 Multisignature setup with Alice and Bob using a central coordinator
+
+1.  The coordinator has published a PSBT that spends from a multi-signature output controlled by Alice and Bob
```

Comment:
```suggestion
1.  The coordinator publishes a PSBT that spends outputs controlled by Alice and Bob's multi-signature.
```

Suggestions: 
- Change step 1 to present tense so it matches the other steps. 
- Since step 2 notes Alice could be signing more than one input, update step 1 to reflect this possibility.

Reply:
This makes sense. Good updates!